docs/install.cmd

```diff
@@ -1,60 +1,98 @@
+echo "      | ##  \__/ /######  /######  /####### /######| ##   | ##/##/######/#### "
+echo "      |  ###### /##__  ##|____  ##/##_____//##__  #|  ## / ##| #| ##_  ##_  ##"
+echo "       \____  #| ##  \ ## /######| ##     | ########\  ## ##/| #| ## \ ## \ ##"
+echo "       /##  \ #| ##  | ##/##__  #| ##     | ##_____/ \  ###/ | #| ## | ## | ##"
+echo "      |  ######| #######|  ######|  ######|  #######  \  #/  | #| ## | ## | ##"
+echo "       \______/| ##____/ \_______/\_______/\_______/   \_/   |__|__/ |__/ |__/"
+echo "               | ##                                                           "
+echo "               | ##                                                           "
+echo "               |__/                                                           "
+echo "                      version : 0.7.0-dev       by : spacevim.org             "
+
+Push-Location ~
+
+$app_name    = "SpaceVim"
+$repo_url    = "https://github.com/SpaceVim/SpaceVim.git"
+$repo_name   = "SpaceVim"
+$repo_path   = "$HOME\.SpaceVim"
+
+echo "==> Starting Testing Procedure..."
+echo ""
+sleep 1
+
+
+echo "==> Testing git"
+git version
+if (!(echo $?)) {
+	echo "[ERROR] Please install git or  make your PATH correctly set! "
+	echo ">>> Ready to Exit......"
+	Pause
+	exit
+} else {
+	echo "[OK] Test successfully. Moving to next..."
+    sleep 1
+}
+
+echo ""
+
+echo "==> Testing vim"
+gvim --version
+if (!(echo $?)) {
+	echo "[WARNING] You didn't install gvim or vim. But intallation can still continue..."
+    echo ""
+	echo "[WARNING] Please install gvim later or  make your PATH correctly set! "
+	Pause
+} else {
+	echo ($(vim --version) -split '\n')[0]
+	echo "[OK] Test successfully. Moving to next..."
+    sleep 1
+}
+
+echo "<== Testing Procedure Completed. Moving to next..."
+sleep 1
+echo ""
+echo ""
+
+if (!(Test-Path "$HOME\.SpaceVim")) {
+    echo "==> Trying to clone $app_name"
+    git clone $repo_url $repo_path
+} else {
+    echo "==> Trying to update $app_name"
+    Push-Location $repo_path
+    git pull origin dev
+}
+
+echo ""
+
+if (!(Test-Path "$HOME\vimfiles")) {
+    cmd /c mklink $HOME\vimfiles $repo_path
+} else {
+    echo "[OK] vimfiles already exists"
+	sleep 1
+}
+
+echo ""
+echo "Almost done!"
+echo "=============================================================================="
+echo "==        Open GVim and it will install the plugins automatically           =="
+echo "=============================================================================="
+echo ""
+echo "That's it. Thanks for installing $app_name. Enjoy!"
+echo ""
+
+Function Pause ($Message = "Press any key to continue . . . ") {
+    if ((Test-Path variable:psISE) -and $psISE) {
+        $Shell = New-Object -ComObject "WScript.Shell"
+        $Button = $Shell.Popup("Click OK to continue.", 0, "Script Paused", 0)
+    } else {     
+        Write-Host -NoNewline $Message
+        [void][System.Console]::ReadKey($true)
+        Write-Host
+    }
+}
+
+Pause
```

Comment:
please use as-if unix format, and remove the NL

Reply:
unix format isn't friendly to windows users when they are trying to look what's inside the `install.cmd` with notepad. It could be messy.